nixos/modules/services/networking/v2raya.nix

```
@@ -6,11 +6,22 @@ with lib;
   options = {
     services.v2raya = {
       enable = options.mkEnableOption "the v2rayA service";
+
+      cliPackage = mkOption {
```

```suggestion
      package = mkOption {
```
Please use mkPackageOption